tests/test_chi_ohare_noise.py

```diff
@@ -0,0 +1,451 @@
+import datetime as dt
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMISSION, COMMITTEE, PASSED, TENTATIVE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ohare_noise import ChiOhareNoiseSpider
+
+spider = ChiOhareNoiseSpider()
+
+freezer = freeze_time("2020-08-22")
+freezer.start()
+
+test_response = file_response(
+    join(dirname(__file__), "files/chi_ohare_noise", "chi_ohare_noise.html"),
+    url=(
+        "https://www.oharenoise.org/about-oncc"
+        + "/oncc-meetings/month.calendar/2019/02/03/-"
+    ),
+)
+
+parsed_items = [item for item in spider.ChiOhareNoiseSubSpider1().parse(test_response)]
+
+freezer.stop()
+
+
+def test_tests():
+    assert len(parsed_items) == 45
+
+
+parsed_sub_items = []
+for i in range(5):
```

Comment:
Ideally we should try not to have more than 2 test files here, and running some logic in `parse` in a separate method that doesn't yield a request could make that easier. That way we could call something like `_parse_detail` without needing to worry about testing pagination